portfolio-project-matching-app/components/Search.js

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react'
+import Button from '../components/Button'
+import Input from '../components/Input'
+
+const Search = ({ onSearch }) => {
```

Comment:
This component was inspired by Josh's inline search bar in the browseProjects wireframe. Although, I'd guess we could find some use for it elsewhere as well.